src/applications/travel-pay/util/constants.js

```diff
@@ -1,10 +1,26 @@
+/**
+ * "Denied", "On hold", and "In manual review" are listed first due to
+ * users desiring to act on those the most. Other can be in alphabetical
+ * order after that.
+ */
 export const CLAIMS_STATUSES = {
```

Comment:
claim status keys taken from the latest [here](https://github.com/department-of-veterans-affairs/btsss-beneficiary-travel-api/blob/develop/azure-resources/apim-configs/apim-frontend-config.eis.devtest.v1.1.json)

Reply:
Just a quick note: the link you referenced is `v1.1` of the API. We are and should be using for the time being `v1`. Could you please double check that these statuses match the `v1` version as well? I don't expect them to be different, but good to call out.

Reply:
Also @hinzed1127, can you remind me the purpose of this mapping object? We can talk about if it's best practice, but currently the claims come back in title case which `Looks Like This`. I noticed that this mapping object has them in sentence case `Which looks like this`.

It seems like a small thing, but I'm worried about the reconciliation between what the backend passes and what the frontend expects.

Reply:
that's a good catch. turns out they were the same between 1.0 and 1.1, but I refactored to just utilize the `vets-api` response and removed the need for this mapping entirely